src/data/chats/chat_entity.rs

```diff
@@ -0,0 +1,38 @@
+//! Hopefully, provisional solution to unify model files and agents in the chat system.
+
+use moly_mofa::MofaAgent;
+use moly_protocol::data::{File, FileID};
+use serde::{Deserialize, Serialize};
+
+/// Identifies either a model file or an agent.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub enum ChatEntityId {
```

Comment:
This type is actually what was called `ChatEntity` in `chat.rs`. Renamed it to `Id` as it's not intended to own the entity itself. Agents are currently enums, so they are identified by themselves, not like files.